installscripts/jazz-terraform-unix-noinstances/scripts/scmpush.sh

```diff
@@ -11,7 +11,13 @@ ns_id_slf=$6
 
 # Config variables
 scm=$7
-jazzbuildmodule=$8
+jenkins_elb=$8
+jenkins_user=$9
+jenkins_password=${10}
+jazzbuildmodule=${11}
+job_url="/project/Trigger_Platform_Services_Build"
```

Comment:
this variable is very specific to gitlabs, so can we name it appropriately?

Reply:
In cooperated the comment